static/js/unscrambleEmail.js

```diff
@@ -0,0 +1,21 @@
+import { unscramble } from "botex";
+
+export function onRouteDidUpdate({ location, previousLocation }) {
+  if (typeof window !== "undefined") {
+    setTimeout(() => {
+      document.body.addEventListener("click", function (event) {
+        if (event.target.matches('a[href="/#contact-email"]')) {
+          event.preventDefault();
+          const contactEmailKey = "CHKXRg4cdMZ0XADF";
+          const obfuscatedContactEmail =
+            "BwC1311181fB4C1sBzmC14162k1dByC112z16132t1o1jBh";
```

Comment:
Let's mode these lines out of the function body. Since they are local in this file, we can use shorter names like `key` (or `botexKey`) and `obfuscatedEmail`.